### PR TITLE
Aait.web.g3.dawit andargachew.profile

### PR DESCRIPTION
**Pull request by dawit-andargachew**
we have added a blog card and it is a reusable component.
Here are the changes we have made
 - components/blog - contains blog car
 - data/myBlog - dummy data used for myblog profile part
 - types/blog - contains the interface need for blog card
 
image

### CODE REVIEW

File: aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          </span>
+        ))}
+      </div>
+
+      <p className="text-[12px]  pb-5  text-[#8E8E8E]">{blog.content}</p>
+
+      <p className="flex justify-between items-center mx-1 mb-2">
+        <span className="flex  gap-x-2">
+          <Image
+            src={"assets/pendingClock.svg"}
```
Comment (kidus101) [inline aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx:42]: Check out image naming conventions.

File: aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx
Comment (kidus101) [inline aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx:1]: Component Naming Convention.

File: aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx (hunk 1 above)
Comment (kidus101) [inline aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx:42]: Image Naming Convention.

File: aait/web/group-3/starter_project_group3/data/myBlogs.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import { BlogModel } from "@/types/Blog/blogTypes";
+
+export const blogsList: BlogModel[] = [
```
Comment (kidus101) [inline aait/web/group-3/starter_project_group3/data/myBlogs.ts:3]: Component Naming Convention.

**Review by kidus101 — COMMENTED** [review]
Focus On Naming Conventions for Components and file name naming for images in the assets folder

File: aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx (hunk 1 above)
Comment (dawit-andargachew) [reply aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx]: resolved

File: aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx
Comment (dawit-andargachew) [reply aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx:1]: resolved and follows the naming convention.

File: aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx (hunk 1 above)
Comment (dawit-andargachew) [reply aait/web/group-3/starter_project_group3/components/Blog/blogCard.tsx]: resolved and follows naming conventions.

**Comment by dawit-andargachew** [comment]
###

**Pull request closed without merging**